src/modules/Task.js

```diff
@@ -18,6 +18,6 @@ export default class Task {
 
   editTask(description, index) {
     this.index = index;
-    DataStore.editTask(description, this.index - 1);
+    DataStore.editTask(description, this.index);
   }
 }
```

Comment:
Kindly add an event listener and method that will make sure that when I click on the checkbox for an item the value of key `isCompleted` is changed to `true` as per project requirements:
![image](https://user-images.githubusercontent.com/38088818/166101485-70744e0b-95ff-45ee-9a5f-9d69fe129990.png)

- Currently when I click on the checkbox to complete an item, the value of `isCompleted` remains `false` on the local storage array.
![image](https://user-images.githubusercontent.com/38088818/166101744-90e81637-53cc-4993-85ce-7b864ed27c9d.png)

<br />
<br />

As a result, when I reload the page all checked items default to unchecked. All checked items should remain checked after I reload the page, and the values of `isCompleted` should remain as `true` for checked items in the local storage after page reload.

Before reload:
![image](https://user-images.githubusercontent.com/38088818/166102003-924849ec-2c52-47c9-b23a-6c66096c4aa6.png)

After reload:
![image](https://user-images.githubusercontent.com/38088818/166102013-5039f3b8-1e54-48e6-bdb8-33a29f320c87.png)

Reply:
Required Changes done.
Merge conflict resolved